index.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html>
+
+<head>
+  <title>Calculator Benchmark</title>
+  <meta charset="utf-8">
+  <meta name="description" content="calculator">
+  <meta name="author" content="Keith Oppel">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100" rel="stylesheet">
+  <link type="text/css" rel="stylesheet" href="main.css">
+</head>
+
+<body>
+  <div class="calculator-container">
+    <div class="calculator-display">
+      <span class="display">0</span>
```

Comment:
According to the spec, the class name should be namespaced under `calculator`, i.e. `calculator-display`.